### PR TITLE
Refresh /iot/intel-iotg

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Don't merge** Currently known release date: Nov 15th, 08:30GMT
- Refresh /iot/intel-iotg based on [copydoc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit#)
- Update url from `/iot/intel-iotg` to `/iot/intel-iot`, as per Felicia's request

## QA

- Go to https://ubuntu-com-12255.demos.haus/download/iot/intel-iot and compare against the copydoc
- Check each link in the table of downloads

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6128

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@petesfrench I would try something like the example below. If you could also add a bit of space above "Are you working on a commercial project?" in the second section that would be good. Thanks.

image

**Comment by petesfrench** [comment]
Thank you @spencerbygraves, I have made those changes.

Can the sections below all remain left aligned? I know sometimes we alternate.
image

**Comment by spencerbygraves** [comment]
@petesfrench
> Can the sections below all remain left aligned?

I think so, but I'd make the titles h2.

Thanks

**Comment by sowasred2012** [comment]
I find the table full of green CTAs overwhelming, like all of them are trying to grab my attention at once. Would something like the first table in this screen still work? image

@spencerbygraves ^

File: templates/download/iot/intel-iot.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    </div>
+    <div class="col-5 u-hide--medium u-hide--small u-align--center">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/efca8490-intel-logo-0.png",
+        alt="",
+        width="300",
+        height="300",
+        hi_def=True,
+        loading="lazy"
```
Comment (sowasred2012) [inline templates/download/iot/intel-iot.html:25]: Appears above the "fold"

```suggestion
        loading="auto"
```

File: templates/download/iot/intel-iot.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      <p>Optimised Linux with unique Intel features</p>
+    </div>
+    <div class="col-6 col-medium-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ceb481d1-10+years.svg",
+        alt="",
+        width="50",
+        height="50",
+        hi_def=True,
+        loading="lazy"
```
Comment (sowasred2012) [inline templates/download/iot/intel-iot.html:67]: Appears above the "fold"

```suggestion
        loading="auto"
```

File: templates/download/iot/intel-iot.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      <p>10+ years of security maintenance and support</p>
+    </div>
+    <div class="col-6 col-medium-3">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/dbd97444-partner-with-us.svg",
+        alt="",
+        width="68",
+        height="50",
+        hi_def=True,
+        loading="lazy"
```
Comment (sowasred2012) [inline templates/download/iot/intel-iot.html:80]: Appears above the "fold"

```suggestion
        loading="auto"
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Couple of minor things, and I'm really not sure about that wall of green CTAs, but looking good otherwise. Was the URL change a request? I don't see where that's documented. We will need to add a redirect.

**Comment by spencerbygraves** [comment]
@sowasred2012 yeah, I think you're right, white buttons would probably be better.

**Review by sowasred2012 — APPROVED** [review]
code & QA :+1:

**Comment by marinacastejon** [comment]
@petesfrench looks good to me except one thing, the Intel product names in the table are not the same as in the copy doc, and there is a comment from Felicia about updating them. If what was done in the demo site is the same as what she requested let me know and I'll update the review label, thanks!

**Comment by petesfrench** [comment]
@marinacastejon This one is ready for re-review

File: templates/download/iot/intel-iot.html [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width u-sv3">
+    <h2>Ubuntu optimised for next-gen Intel SoCs</h2>
+    <p>Intel's portfolio of edge-ready compute and connectivity technologies make it easy to deploy edge applications quickly. Enhanced for IoT, they enable processing at the edge to get critical insights and business value from your data with compute resources where you need them most.</p>
+    <p><strong>Download Ubuntu for your Intel hardware.</strong> Below you will find links for the production-grade family of Ubuntu images, along with release notes and installation instructions.</p>
+    <p><a href="https://assets.ubuntu.com/v1/642fcdac-December_14-Release+Notes.pdf">Check out the release notes&nbsp;&rsaquo;</a></p>
+    <table class="p-table--mobile-card" aria-label="Ubuntu 22.04 for intel downloads">
+      <thead>
+        <tr>
+          <th></th>
```
Comment (sowasred2012) [inline templates/download/iot/intel-iot.html:105]:
```suggestion
          <th><span class="u-off-screen">Hardware type</span></th>
```

File: templates/download/iot/intel-iot.html [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+          <td class="u-align--center" data-heading="Ubuntu Desktop 22.04"><p><a href="http://cdimage.ubuntu.com/jammy/daily-live/20221020/jammy-desktop-amd64+intel-iot.iso" class="p-button is-dense">Download</a></p></td>
+          <td class="u-align--center" data-heading="Ubuntu Server 22.04"><p><a href="https://cdimage.ubuntu.com/ubuntu-server/jammy/daily-live/20221013.2/jammy-live-server-amd64+intel-iot.iso" class="p-button is-dense">Download</a></p></td>
+        </tr>                
+      </tbody>
+    </table>
+
+    <table class="p-table--mobile-card" aria-label="Ubuntu 20.04 for intel downloads">
+      <thead>
+        <tr>
+          <th></th>
```
Comment (sowasred2012) [inline templates/download/iot/intel-iot.html:142]:
```suggestion
          <th><span class="u-off-screen">Hardware type</span></th>
```

**Review by sowasred2012 — APPROVED** [review]
A couple of a11y improvements, but looks good otherwise.

**Comment by marinacastejon** [comment]
LGTM, thanks Pete

**Pull request merged**